### ISSUE TITLE
The world is not loading

### ISSUE THREAD

**Issue by Neubulae** [KenRouKoro/C3H6N6O6#15]
![图片](https://user-images.githubusercontent.com/24774842/194721865-4e9a6493-9181-470c-940a-2a8e289bbb3c.png)
I have VMP installed, I guess that's a culprit?

**Comment by KenRouKoro** [comment]
Can you provide your modlist?

**Comment by Neubulae** [comment]
https://0x0.st/ovGG.txt

**Comment by wicivo** [comment]
> ![图片](https://user-images.githubusercontent.com/24774842/194721865-4e9a6493-9181-470c-940a-2a8e289bbb3c.png) У меня установлен VMP, я думаю, что это виновник?

да, VMP не совместим с  этим модом. Лично у меня они вообще вместе не запускаются
